libc/utils/docgen/api.yml

```diff
@@ -0,0 +1,107 @@
+ctype.h:
+  functions:
+    isalnum:
+      returns: int
+      params:
+        - int
+    isalpha:
+      returns: int
+      params:
+        - int
+    isblank:
+      returns: int
+      params:
+        - int
+  # TODO: fill me out!
+fenv.h:
+  macros:
+    - __STDC_VERSION_FENV_H__
+  functions:
+    feclearexcept:
+      returns: int
+      params:
+        - int
+      defined: '7.6.4.1'
```

Comment:
Perhaps `defined` should be a list. For example, if something is specified by both C and POSIX, maybe it makes sense to refer to BOTH sections?

The C standard can't be hyperlinked to AFAIK unfortunately, but POSIX can be IIRC. Maybe supporting hyperlinking would be nice.